orc8r/cloud/go/obsidian/swagger/spec_servicer.go

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger
+
+import (
+	"context"
+	"io/ioutil"
+
+	"magma/orc8r/cloud/go/obsidian/swagger/protos"
+	"magma/orc8r/lib/go/service/config"
+
+	"github.com/golang/glog"
+)
+
+type specServicer struct {
+	spec string
+}
+
+func NewSpecServicerFromFile(service string) protos.SwaggerSpecServer {
+	path := config.GetSpecPath(service)
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		// We are swallowing this error because a singular service spec failure
+		// should not down the entire Swagger UI
```

Comment:
This isn't quite the reason we're swallowing this error -- we'll handle not borking the Swagger UI at the polling side (client side). This side (servicer side) we're swallowing the error because the service should continue running even if it can't find its Swagger spec file

Reply:
Ah I see, that definitely seems like a better reason.